replica/rest/model.go

```diff
@@ -43,6 +43,7 @@ type RebuildingInput struct {
 type SnapshotInput struct {
 	client.Resource
 	Name        string `json:"name"`
```

Comment:
For creating the snapshot do we need to pass the headname as well or controller will take care of this?

Reply:
Controller will take care of this